Editor/ChannelPackerTool.cs

```diff
@@ -41,7 +39,11 @@ public static void OpenEditorWindow()
         private void CreateGUI()
         {
             VisualElement root = rootVisualElement;
-            var visualTree = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>("Assets/ChannelPackerTool/Resources/UI Documents/ChannelPackerToolWindow.uxml");
+            string[] paths = AssetDatabase.FindAssets("t:VisualTreeAsset ChannelPackerToolWindow");
+            if (paths.Length != 1)
+                throw new Exception($"Unexpected number of ChannelPackerToolWindow VisualTreeAssets found: expected=1, got={paths.Length}");
+            
+            var visualTree = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(AssetDatabase.GUIDToAssetPath(paths[0]));
             VisualElement tree = visualTree.Instantiate();
             root.Add(tree);
 
```

Comment:
Everything else below just moves `Resources` under `Editor`. Resources is a special magic folder name and anything in a resource folder will be included in the built game, which you don't really want for an editor plugin. For example the Transparency500.png shouldn't be included in the built game because it would just add bloat for something only used in the editor. 

Moving Resources under Editor means it will be available as an editor resource but stripped from the built game.